### ISSUE TITLE
[MEDIUM] np.log10 without +1 offset in some normalization paths

### ISSUE THREAD

**Issue by Maxelee** [Maxelee/vdm_BIND#16]
## Description

Some normalization code paths use `np.log10(field)` instead of `np.log10(field + 1)`. Since astrophysical fields can contain zeros (empty voxels), this produces `-inf` values that propagate through the network.

The canonical normalization documented in `copilot-instructions.md` is:
```python
field_norm = np.log10(field + 1)  # Correct: +1 offset prevents log(0)
```

But some code paths omit the `+1`:
```python
field_norm = np.log10(field)  # Bug: log10(0) = -inf
```

## Impact

- `-inf` values in input tensors cause NaN losses during training
- May only affect certain fields (e.g., stellar mass which has many zero voxels)
- Inconsistent normalization between training and inference produces wrong results

## Fix

Audit all `np.log10` and `torch.log10` calls to ensure they use the `+1` offset consistently:
```python
field_norm = np.log10(field + 1)
```

## Files
Multiple files in `vdm/` and `bind/` — search for `log10` calls
